api-reference/10 UI Components/dxDateRangeBox/1 Configuration/endDate.md

```diff
@@ -5,7 +5,24 @@ default: null
 ---
 ---
 ##### shortDescription
-Specifies the range's end date.
+Specifies the range's end date. 
 
 ---
-We recommend that you specify dates in <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">UTC ISO 8601</a>.
+You can specify the end date with any of the following formats:
+
+- Date  
+ Specifies the date directly.
```

Comment:
Can we say this? 

```suggestion
 Specifies a date object.
```

Reply:
Yes. Users can specify a date as the Date() object. I agree, that "directly" can be misleading